mods/voidmulti.sc2mod/base.sc2data/gamedata/behaviordata.xml

```diff
@@ -9,19 +9,28 @@
     <CBehaviorBuff id="AdeptPhaseShift">
         <Modification MoveSpeedBonus="0"/>
     </CBehaviorBuff>
+    <CBehaviorBuff id="AdeptPhaseShiftCaster">
+        <Modification>
+            <StateFlags index="SuppressPassenger" value="0"/>
```

Comment:
Adepts can now be ordered to load into a Warp Prism while shading, automatically cancelling the Shade

This line allows being loaded into transports